src/solver/application/application.cpp

```diff
@@ -451,7 +437,7 @@ void Application::resetLogFilename() const
     }
 
     logfile /= "solver-";               // append the filename
-    logfile += timeToString() + ".log"; // complete filename with timestamp and extension
+    logfile += FormattedTime("%Y%m%d-%H%M%S") + ".log"; // complete filename with timestamp and extension
```

Comment:
i'm surprised that this work? fs::path+std::string 😮

Reply:
`FormattedTime("%Y%m%d-%H%M%S")` is a **std::string**.
So `FormattedTime("%Y%m%d-%H%M%S") + ".log"` is a std::string as well.
Then we use operator **+=** of **fs::path**, which takes a **std::string** as argument, and append the path with this string

Reply:
thanks, i didn't notice the += ope